tools/topology/topology2/include/components/dcblock/default.conf

```diff
@@ -0,0 +1,12 @@
+# Exported with script example_dcblock.m 27-Sep-2023
```

Comment:
but what about the cmd line args used to generate this and other blobs ? i.e. how would someone else reproduce this ?

Reply:
Though I guess you meant how to launch the blobs generator from shell command line. I can add that as well.

Reply:
It would be then `cd tools/tune/dcblock; octave --no-window-system example_dcblock.m` .